cmd/sealer/cmd/image/build.go

```diff
@@ -315,8 +322,6 @@ func applyRegistryToImage(engine imageengine.Interface, imageID string, platform
 		return "", nil, errors.Wrapf(err, "failed to create working container, imageid: %s", imageID)
 	}
 
-	var containerImageList []*v12.ContainerImage
-
 	// download container image from `imageList`
 	if buildFlags.ImageList != "" && osi.IsFileExist(buildFlags.ImageList) {
 		images, err := osi.NewFileReader(buildFlags.ImageList).ReadLines()
```

Comment:
add  the validation of the image format for image-list

Reply:
> add the validation of the image format for image-list

added